capstone-app/src/app/modules/photos/assets/aquarium.json

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
Optional: Consider renaming this file (or the directory) to make it clear that this is fake, test data.

Reply:
Same comment for other pieces of fake data